runtime/near-vm-logic/src/logic.rs

```diff
@@ -639,6 +640,29 @@ impl<'a> VMLogic<'a> {
         Ok(self.context.block_height)
     }
 
+    /// Gets a block given a height (up to 256)
+    ///
+    /// # Cost
+    ///
+    /// `base`
+    #[cfg(feature = "protocol_feature_get_block_by_hash")]
+    pub fn get_block_hash_by_height(
```

Comment:
On naming, of course, this is subjective, but personally I would just call the function `block_hash`.

First, [it is conventional in Rust](`block_hash_next`) to name getters without the `get_`. Second, the `by_height` part is a bit redundant redundant, as the type of argument makes it clear as well.

Given that this becomes part of the smart contract runtime API, finding a good name is important. So if you (or anyone else) disagrees with my suggestion of just calling it `block_hash`, it should probably be discussed on the NEP.

Reply:
I don't think that Rust's naming conventions should have any bearing here: we define language-agnostic WASM ABI, the implementation language of nearcore itself is completely irrelevant here. We should do what makes sense for WASM. 

And I think the primary guideline should be consistency. We already have `validator_stake` and `promise_result`, so we shouldn't have a `get` prefix. 

So, I agree with conclusion, but for a different reason :)